driver-sync/src/main/com/mongodb/client/ListSearchIndexesIterable.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client;
+
+import com.mongodb.ExplainVerbosity;
+import com.mongodb.client.model.Collation;
+import com.mongodb.lang.Nullable;
+import org.bson.BsonValue;
+import org.bson.Document;
+
+import java.util.concurrent.TimeUnit;
+/**
+ * Iterable for listing Atlas Search indexes.
+ * This interface contains aggregate options and that of applied to {@code $listSearchIndexes} operation.
+ *
+ * @param <TResult> The type of the result.
+ * @mongodb.driver.manual reference/command/aggregate/ Aggregation
```

Comment:
Update manual reference to atlas search.

Reply:
I think it could be updated to [ reference/operator/aggregation/listSearchIndexes ListSearchIndexes](https://www.mongodb.com/docs/upcoming/reference/operator/aggregation/listSearchIndexes). The page also provides necessary links to further explore Atlas Search Indexes